src/pages/Coins/Coins.js

```diff
@@ -32,13 +32,13 @@ const Coins = ({main, getChartData, getCoins, coins, getMoreCoins, sortItems, so
     }
   }, [location, changePath]);
   useEffect(() => {
-    if(window.innerWidth > 1020){
+    if(window.innerWidth > 900){
       setScreen(true)
     } else {
       setScreen(false);
     }
     const updateMedia = () => {
-      if (window.innerWidth > 1020) {
+      if (window.innerWidth > 900) {
```

Comment:
i see you're using this screen size functionality in multiple spots, consider making a reusable hook